res/skins/QMLDemo/Deck.qml

```diff
@@ -0,0 +1,263 @@
+import Mixxx 0.1 as Mixxx
+import Mixxx.Controls 0.1 as MixxxControls
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.11
+
+Item {
+    id: root
+
+    required property string group
+
+    ColumnLayout {
+        anchors.fill: parent
+
+        RowLayout {
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+
+            ColumnLayout {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+
+                Rectangle {
+                    id: trackInfo
+
+                    Layout.fillWidth: true
+                    height: 60
+                    color: "#121213"
+
+                    Rectangle {
+                        id: coverArt
+
+                        anchors.top: parent.top
+                        anchors.left: parent.left
+                        anchors.bottom: parent.bottom
```

Comment:
```suggestion
                        anchors.verticalCenter: parent.verticalCenter
                        anchors.left: parent.left
```
can you try this out? not sure if it has the same result though...

Reply:
No, this centers the item, but it won't show up because it has no height. The current code sets the height to that of the parent.

Reply:
Ok good to know, so the anchor doubles as a responsive layout guide.